msal/src/MSAL.Common.props

```diff
@@ -0,0 +1,204 @@
+<Project>
```

Comment:
I moved most of the MSAL.csproj into this file, which will be used by MSAL.ref.csproj as well. I will point out differences, as csproj files are a joy to read.